test/e2e/cli_test.go

```diff
@@ -0,0 +1,95 @@
+package e2e
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	"github.com/argoproj/argo/test/e2e/fixtures"
+)
+
+type CLISuite struct {
+	fixtures.E2ESuite
+}
+
+func (s *CLISuite) BeforeTest(suiteName, testName string) {
+	s.E2ESuite.BeforeTest(suiteName, testName)
+	_ = os.Setenv("ARGO_SERVER", "localhost:2746")
+}
+
+func (s *CLISuite) AfterTest(suiteName, testName string) {
+	s.E2ESuite.AfterTest(suiteName, testName)
+	_ = os.Unsetenv("ARGO_SERVER")
+}
+
+func argo(args ...string) (string, error) {
+	output, err := exec.Command("../../dist/argo", args...).CombinedOutput()
+	return string(output), err
+}
+
+func (s *CLISuite) TestCompletion() {
+	output, err := argo("completion", "bash")
+	s.Assert().NoError(err)
+	s.Assert().Contains(output, "bash completion for argo")
+}
+
+func (s *CLISuite) TestCore() {
```

Comment:
@sarabala1979 @simster7 we should add additional tests for change CLI commands to this file to avoid regression- do you think I can ask you to do this when you make change please? They only need to be basic acceptance tests, see argo_server_suite.go for examples that use the API.